src/components/menus/directory-tree-contextual-menu.tsx

```diff
@@ -48,13 +48,13 @@ interface DirectoryTreeContextualMenuProps {
     onClose: (e: unknown, nextSelectedDirectoryId?: string | null) => void;
     openDialog: string;
     setOpenDialog: (dialogId: string) => void;
-    isEmptyDirectory: boolean;
+    hideSomeMenuOptions: boolean;
```

Comment:
i would suggest to rename this props to 'isReadOnlyMode' or 'hideModifyDirectoryOptions"

Reply:
I change it to restrictMenuItems.